### DISCUSSION TITLE
Easy to use u8g2 conversion tools

### DISCUSSION THREAD

**Discussion by stonez56** [olikraus/u8g2#2068] [Show and tell]
@olikraus  
I have created a web tool ([Stonez56](https://kidsgo.net/u8g2)) to ease the process of creating unifont codes. 
The font base I used is the bdf/unifont.bdf  and it seemed converted Chinese, Japanese, and Korean quite well. (see images below)

![lcd1_result](https://user-images.githubusercontent.com/5953052/211127993-e2ffc835-76db-4120-9f61-7f85eafaab58.png) ![lcd2_result](https://user-images.githubusercontent.com/5953052/211127995-1d1fb886-b97b-41fb-aaf1-58ab700b1319.png) ![lcd3_result](https://user-images.githubusercontent.com/5953052/211127996-9daa575b-511b-4b71-8013-30ae0b61d959.png) ![lcd4_result](https://user-images.githubusercontent.com/5953052/211127997-38201be5-949d-4c73-a3b9-8ab842958001.png)

Currently, unifonts I converted are 16x16 pixels for Chinese, Japanese, and Korean. 
Someone asked, can they change the font to larger sizes?  
Any recommendation for this to happen?

**Comment by olikraus** [comment]
Your question is not clear to me. In general bitmap fonts like BDF can't be changed in size automatically.

**Reply by stonez56** [reply]
Thanks, that's exactly my question.  I understand now, it's not possible to change the font size.
And I was amazed by how many characters in unifont.bdf contained.

**Reply by olikraus** [reply]
There those "wenquanyi" bitmap fonts, which are available in different sizes. They are part of u8g2. The original source is mentioned here: https://github.com/olikraus/u8g2/issues/455

**Reply by stonez56** [reply]
Thanks for the info. I will try wenquanyi bitmap fonts for Simplified Chinese.
